src/components/index.js

```diff
@@ -1,2 +1,3 @@
 export * from './Button'
 export * from './PageTitle'
+export * from './Navbar'
```

Comment:
Layout elements should be in `src/layout`. This is worth to not mix them with common elements